src/components/Apply/index.jsx

```diff
@@ -6,17 +6,17 @@ import {
   authenticate,
   getApplication,
   getRoles,
-  isAuthenticated
+  isAuthenticated,
```

Comment:
not a fan of trailing commas, but feel free to keep them in if you like them

Reply:
yeah, i like trailing commas because it's nicer when you want to add more things